api/src/main/java/org/openmrs/FormField.java

```diff
@@ -22,32 +34,54 @@
  * @see org.openmrs.Form
  * @see org.openmrs.Field
  */
+@Entity
+@Table(name = "form_field")
+@AttributeOverride(name = "name", column = @Column(name = "name", nullable = true))
+@AttributeOverride(name = "retired", column = @Column(name = "retired", nullable = true, columnDefinition = "boolean default false"))
 public class FormField extends BaseChangeableOpenmrsMetadata implements java.io.Serializable, Comparable<FormField> {
-	
+
 	public static final long serialVersionUID = 3456L;
-	
```

Comment:
@alinmihaila99   Thanks for working on this,though it came out with lots of code formats